src/components/App.tsx

```diff
@@ -74,6 +74,7 @@ function App() {
                   <SearchView
                     toggleColorMode={toggleColorMode}
                     protocolTheme={protocolTheme}
+                    title='PerfCompare - Search'
```

Comment:
Probably these strings should be part of the `resources/Strings.tsx` instead of hardcoded.